app/test_main.py

```diff
@@ -1,3 +1,23 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,result",
+    [
+        (0, 0, [0, 0]),
+        (20, 20, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 28, [2, 2]),
+        (28, 28, [3, 2]),
+        (28, 29, [3, 3]),
+        (100, 100, [21, 17])
+    ]
```

Comment:
The current test cases cover a range of typical inputs, but the checklist suggests testing edge cases such as negative numbers, zero, and very large numbers. Consider adding tests for these scenarios to ensure the function handles all possible inputs correctly.